public/fake_data_for_autocomplete.txt

```diff
@@ -0,0 +1 @@
+{"theaters":{"Blue_Barn_Theater":{"To_Kill_A_Mockingbird":{"Deb_Kelly":["Maudie_Atkinson"],"Jarrell_Roach":["Tom_Robinson"],"Lori_Obradovich":["director"],"John_Carlson":["Atticus_Finch"]}},"Bellevue_Little_Theater":{"someShow":{"person":["role"]}},"The_Rose_Theater":{"Honk!":{"person":["role"]}},"Wilson_Performing_Arts_Center":{"Never_Too_Late":{"person":["role"]},"another_show":{"person":["role"]}}}}
```

Comment:
Something to be mindful of: The JSON response doesn't _need_ to have underscores like this. Instead of "Blue_Barn_Theater", expect "Blue Barn Theater". Not sure that'll make a difference for you, but I thought I'd mention it.